tests/unit/splinterdb_stress_test.c

```diff
@@ -138,6 +139,57 @@ CTEST2(splinterdb_stress, test_naive_range_delete)
    }
 }
 
+/*
+ * Test case that inserts large # of KV-pairs, and goes into a code path
+ * reported by issue# 458, tripping a debug assert. This test case also
+ * triggered the failure(s) reported by issue # 545.
+ */
+CTEST2(splinterdb_stress, test_issue_458_mini_destroy_unused_debug_assert)
+{
+   char key_data[TEST_KEY_SIZE];
+   char val_data[TEST_VALUE_SIZE];
+
+   uint64 test_start_time = platform_get_timestamp();
+
+   for (uint64 ictr = 0, jctr = 0; ictr < 100; ictr++) {
+
+      uint64 start_time = platform_get_timestamp();
+
+      for (jctr = 0; jctr < MILLION; jctr++) {
+
+         uint64 id = (ictr * MILLION) + jctr;
+         snprintf(key_data, sizeof(key_data), "%lu", id);
+         snprintf(val_data, sizeof(val_data), "Row-%lu", id);
+
+         slice key = slice_create(strlen(key_data), key_data);
+         slice val = slice_create(strlen(val_data), val_data);
```

Comment:
This seems to trigger the issue. If you use `TEST_KEY_SIZE` and `TEST_VALUE_SIZE`, you will not be able to repro this problem.